data/data-pipeline/data_pipeline/score/score.py

```diff
@@ -0,0 +1,14 @@
+import pandas as pd
+import data_pipeline.score.field_names as FN
```

Comment:
nit: i generally prefer being verbose and clear over concise with room for misinterpretation, and `FN` is throwing me off since `fn` is often used to mean `function` (that's what I thought this meant at first). 

Any reason we can't just leave it at `import data_pipeline.score.field_names` and reference, e.g., `field_names. POVERTY_PERCENTILE_FIELD`?

Reply:
Also maybe we just import this directly in the child classes, and not here in the parent?

Reply:
done!